util/dockerfiles/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+export CHPL_TARGET_ARCH=`$CHPL_HOME/util/chplenv/chpl_arch.py --target`
```

Comment:
There is a `chpl_bin_subdir.py` that provides the entire bin subdir, and we should use that instead. Should also use `--host` instead of target since this is for the compiler which is built for the host architecture.

Reply:
Done!
Tested and worked fine.